src/config/lib/app.js

```diff
@@ -1,11 +1,11 @@
 module.exports.start = () => {
-  const app = require("./express")();
+	require("dotenv").config();
```

Comment:
keep this line in src/config/index.js file at first line of initEnvironmentVariables function. Thus u can avoid to keep this line in multiple places